<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCpc</class>
 <widget class="QSubDialogBase" name="QCpc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cpc聚类</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://www.cnblogs.com/ironstark/p/5027269.html&quot;&gt;&lt;span style=&quot; font-size:18pt; text-decoration: underline; color:#aa007f;&quot;&gt;点击转到引用原文&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_25">
     <item row="1" column="1">
      <widget class="QLineEdit" name="max_lineEdit_23"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_47">
       <property name="text">
        <string>粒子距离（double）：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="min_lineEdit_23"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_48">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>晶核距离（double）：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_49">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout_26">
     <item row="0" column="0">
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>颜色容差（double）：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="min_lineEdit_24"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_50">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="max_lineEdit_24"/>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_51">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>最大数量的削减（int）：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_24">
     <item row="1" column="2">
      <spacer name="horizontalSpacer_46">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_47">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="min_lineEdit_22"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_45">
       <property name="text">
        <string>最小切割数量（int）：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="max_lineEdit_22"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_46">
       <property name="text">
        <string>建议切割的最小分数必须达到被执行的目标（double）：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_23">
     <item row="1" column="1">
      <widget class="QLineEdit" name="max_lineEdit_21"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>空间权重（double）：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="min_lineEdit_21"/>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_44">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>向量权重（double）：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_45">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>应用</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QSubDialogBase</class>
   <extends>QWidget</extends>
   <header>qsubdialogbase.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QCpc</receiver>
   <slot>OnApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>395</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnApply()</slot>
 </slots>
</ui>
